shared/js/otSdkDriver.js

```diff
@@ -622,6 +624,7 @@ loop.OTSdkDriver = (function() {
         srcMediaElement: sdkSubscriberVideo
       }));
 
+      this._notifyMetricsEvent("Session.subscribeCompleted");
```

Comment:
And even if there was an issue, it'd be better not to log the subscribeCompleted here - as an issue potentially means the subscribe hasn't fully completed anyway.

In any case, I don't think it matters much where this is placed, so long as we log it.